build.gradle

```diff
@@ -127,13 +127,13 @@ test {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:2.3.+')
-    api 'com.exactpro.th2:cradle-core'
+    api platform('com.exactpro.th2:bom:2.9.1')
+    api 'com.exactpro.th2:cradle-core:2.4.0'
```

Comment:
Maybe we should extract the cradle version to a variable?

Reply:
I think no because this project has only one artifact related to cradle

Reply:
And what about `implementation 'com.exactpro.th2:cradle-cassandra:2.4.0'`?

Reply:
O, skipped it, thanks